issues.go

```diff
@@ -46,6 +46,7 @@ type Issue struct {
 		ID          int        `json:"id"`
 		Title       string     `json:"title"`
 		Description string     `json:"description"`
+		StartDate   string     `json:"start_date"`
```

Comment:
I think it better if we delete this inline struct and just use the `Milestone` struct here?

```
<snip>
  Milestone *Milestone `json:"milestone"`
<snip>
```